lib/ansiblelint/skip_utils.py

```diff
@@ -0,0 +1,163 @@
+# (c) 2012-2019, Ansible by Red Hat
```

Comment:
AFAIK the year should be of content creation/update. Which is 2019 for Ansible. Do we have anything written by the original author here? If yes, crediting them is also needed.
```suggestion
# (c) 2019, Ansible by Red Hat
```